fastavro/write.py

```diff
@@ -11,7 +12,7 @@
 Writer = _write.Writer
 json_writer = json_write.json_writer
 schemaless_writer = _write.schemaless_writer
-LOGICAL_WRITERS = _write.LOGICAL_WRITERS
+LOGICAL_WRITERS = logical_writers.LOGICAL_WRITERS
```

Comment:
I imported it also here to keep the interface of `write` untouched.

Reply:
Makes sense 👍